styles/reset.css

```
@@ -0,0 +1,17 @@
+* {
+    box-sizing: border-box;
+    font-family: Pretendard;
```

💊 제안
아래에서 html에 font-family를 설정해주셨으니 이렇게 모든 태그에 font-family 값을 Pretendard로 주시는 것보다
아래처럼 작성하여 상속받게 하는 것이 더 명시적이고 중복을 줄일 수 있을 것 같습니다~

```suggestion
    font-family: inherit;
```